.github/ISSUE_TEMPLATE/graduation.md

```diff
@@ -0,0 +1,34 @@
+# Application for Graduation
+- [ ] Have committers from at least two organizations.
+- [ ] Have achieved and maintained a Core Infrastructure Initiative Best Practices Badge.
+- [ ] Have completed an independent and third party security audit with results published of similar scope and quality as this example which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
```

Comment:
is there an intended link to the example that is missing?

Reply:
https://github.com/envoyproxy/envoy#security-audit is in the .md file